cvmfs/cvmfs.cc

```diff
@@ -1468,45 +1468,63 @@ static void cvmfs_statfs(fuse_req_t req, fuse_ino_t ino) {
            uint64_t(ino));
 
   // If we return 0 it will cause the fs to be ignored in "df"
```

Comment:
Let's move this comment up, before the function signature. Otherwise one might confuse it with a comment related to `TraceInode`.